account_invoice_triple_discount/migrations/16.0.2.0.0/pre-migrate.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from openupgradelib import openupgrade
+
+
+def migrate_discount_to_discount1(env):
```

Comment:
This function should be called in the post-init hook, once the module is installed.
Otherwise, database will be corrupted, because discount1 = 0, even if discount != 0. Don't you think ?